src/Makefile

```diff
@@ -1638,16 +1614,9 @@ LIBS_TEST=-lcmocka
 # LIBS_TEST = -lcheck -lm -lrt -lsubunit
 
 
-ifneq (,$(filter ${TEST},YES yes y Y 1))
-	OSSEC_CFLAGS+=${CFLAGS_TEST}
-	# OSSEC_CFLAGS+=${CFLAGS_TEST} -Itests
-	OSSEC_LDFLAGS+=${CFLAGS_TEST}
-	OSSEC_LIBS+=${LIBS_TEST}
-ifeq (${TARGET},winagent)
-	WINDOWS_UNIT_TEST_WRAPPERS:=${wrappers_syscheck_o}
-	WINDOWS_UNIT_TEST_WRAPPERS+=${wrappers_shared_o}
-endif
-endif #TEST
+
+unit_tests/wrappers/%.o: unit_tests/wrappers/%.c
```

Comment:
this is adapting to how is the master branch: https://github.com/wazuh/wazuh/blob/ff962364251c1fa38468bc750c8431ced9254b84/src/Makefile#L1603